### PR TITLE
Sublime

### PR DESCRIPTION
**Pull request by nexus-uw**
note: based off https://github.com/rangle/ngcourse/pull/55

could @yuri or @SethDavenport review?

### CODE REVIEW

File: handout/18-gulp.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+# Part 18: [gulp](https://github.com/gulpjs/gulp), _The Streaming Build System_
```
Comment (SethDavenport) [inline handout/18-gulp.md]: We already have a section 18 - integration testing.  Not sure if this should come before or after  - @yuri i?

File: handout/18-gulp.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+
+At its simplest, gulp is a build system and collection of plugins using [NodeJS streams](https://github.com/substack/stream-handbook#introduction). The idea is to read in a collection files, pass them through a series of modules, and then finally write out the result.
+
+## Why use gulp?
+
+ - __Avalaible Plugins__: By having modules do one thing well, gulp allows for easily reused modules to be added, reordered and removed without effecting those up/downstream. Leaving a flexible and versatile collection of jobs.
```
Comment (SethDavenport) [inline handout/18-gulp.md]: s/Avalaible/Available/

Comment (SethDavenport) [inline handout/18-gulp.md]: s/effecting/affecting/

Comment (SethDavenport) [inline handout/18-gulp.md]: Sentence fragment.  How about: 'This leaves a flexible and versatile collection of jobs.'

File: handout/18-gulp.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+- __Tasks Are Easily Chained Together__: The use of streams is highly agreeable when trying to build a web app for a production environment where a series of improvements (minification, concatenation, etc )  and checks (linting, unit tests, etc) need be preformed and then outputted to a build folder. Chaining together the different modules together to take in the source files, preform an improvement/transformation, and then output the resulting build to a build folder, will create a repeatable automated build.
```
Comment (SethDavenport) [inline handout/18-gulp.md]: s/etc )/etc.)/

Comment (SethDavenport) [inline handout/18-gulp.md]: s/preformed/performed/

File: handout/19-sublime.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
+# Part 19: [Sublime Text](http://www.sublimetext.com/)
```
Comment (SethDavenport) [inline handout/19-sublime.md]: We already have a module 19 - unit testing controllers and directives.  I think this module should probably be #21, and the gulp module should be #20.

**Comment by nexus-uw** [comment]
@SethDavenport I have fixed the typos/errors you found here on my other PR for gulp
@yuri , as per your slack messages, the gulp example file has been broken apart and the explanations have been expanded

**Comment by yuri** [comment]
@nexus-uw Needs a rebase.

**Comment by nexus-uw** [comment]
@yuri ready to merge now.

**Pull request merged**